src/pages/workspace/accounting/netsuite/import/NetSuiteImportPage.tsx

```diff
@@ -0,0 +1,108 @@
+
+function NetSuiteImportPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '-1';
+    const config = policy?.connections?.netsuite?.options.config;
+    const importFields = CONST.NETSUITE_CONFIG.IMPORT_FIELDS;
+    const importCustomFields = CONST.NETSUITE_CONFIG.IMPORT_CUSTOM_FIELDS;
+
+    return (
+        <ConnectionLayout
+            displayName={NetSuiteImportPage.displayName}
+            headerTitle="workspace.accounting.import"
+            headerSubtitle={config?.subsidiary ?? ''}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            contentContainerStyle={[styles.pb2]}
+            titleStyle={styles.ph5}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.NETSUITE}
+        >
+            <View style={[styles.flex1, styles.ph5, styles.mb4]}>
+                <ToggleSettingOptionRow
+                    title={translate('workspace.netsuite.import.expenseCategories')}
+                    subtitle={translate('workspace.netsuite.import.expenseCategoriesDescription')}
+                    shouldPlaceSubtitleBelowSwitch
+                    isActive
+                    disabled
+                    switchAccessibilityLabel={translate('workspace.netsuite.import.expenseCategories')}
+                    onToggle={() => {}}
+                />
+            </View>
+
+            <View style={styles.mb4}>
+                {importFields.map((importField) => (
+                    <OfflineWithFeedback
+                        key={importField}
+                        errors={ErrorUtils.getLatestErrorField(config ?? {}, importField)}
+                        errorRowStyles={[styles.ph5, styles.mt2, styles.mb4]}
+                        onClose={() => Policy.clearNetSuiteErrorField(policyID, importField)}
+                    >
+                        <MenuItemWithTopDescription
+                            description={translate(`workspace.netsuite.import.importFields.${importField}`)}
+                            title={translate(`workspace.accounting.importTypes.${config?.syncOptions?.mapping?.[importField] ?? CONST.INTEGRATION_ENTITY_MAP_TYPES.NETSUITE_DEFAULT}`)}
+                            shouldShowRightIcon
+                            onPress={() => {
+                                // TODO: Navigation will be handled in future PRs
+                            }}
+                            brickRoadIndicator={config?.errorFields?.[importField] ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+                        />
+                    </OfflineWithFeedback>
+                ))}
+            </View>
+
+            <View style={[styles.flex1, styles.ph5, styles.mb4]}>
+                <ToggleSettingOptionRow
```

Comment:
I think we have a condition on visibility for Tax switch. But I am waiting for Export method to be merged. It has some helper methods that I can use. Hence, I'll fix this in future PRs.

Reply:
I've added a condition now.